core/bin/external_node/src/tests.rs

```diff
@@ -94,6 +99,34 @@ fn expected_health_components(components: &ComponentsToRun) -> Vec<&'static str>
     output
 }
 
+fn mock_eth_client(diamond_proxy_addr: Address) -> MockEthereum {
+    MockEthereum::default().with_call_handler(move |call, _| {
+        tracing::info!("L1 call: {call:?}");
+        if call.to == Some(diamond_proxy_addr) {
+            let call_signature = &call.data.as_ref().unwrap().0[..4];
+            let contract = zksync_contracts::hyperchain_contract();
+            let pricing_mode_sig = contract
+                .function("getPubdataPricingMode")
```

Comment:
is there no way of hardcoding those method names here (just for tests)? Perhaps there are some already defined constants that can be used instead?